src/styles/blocks/reset.scss

```
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+}
+
+.card__button {
+  text-decoration: none;
+}
+
+p {
+  margin: 0;
+}
```

Keep styles for tags above classes.

```suggestion
p {
  margin: 0;
}

.card__button {
  text-decoration: none;
}
```